### PR TITLE
Adding a batch of new questions

### PR DESCRIPTION
**Pull request by MrWolfbear**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3060,4 +3071,113 @@ _[⬆ Back to Top](#table-of-contents)_
 - [ ] Demonstrating code is well structured and cleanly implemented.
 - [x] Measuring branching structures and nesting levels in code.
 
+_[⬆ Back to Top](#table-of-contents)_
+
+### The Developers should have all the skills needed to: (choose the best answer)
+
+- [ ] Complete the project as estimated when the date and cost are committed to the Product Owner.
+- [x] Turn the Product Backlog items they select into an Increment of useful and valuable product functionality.
+- [ ] Do all of the development work, except for specialized testing that requires additional tools and environments.
```
Comment (MrWolfbear) [inline README.md:3080]: image

File: README.md [hunk 2]
```diff
@@ -3060,4 +3071,113 @@ _[⬆ Back to Top](#table-of-contents)_
...
+
+
+_[⬆ Back to Top](#table-of-contents)_
+
+### Which statement best describes Scrum? (choose the best answer)
+
+- [x] A framework for creating complex products in complex environments.
+- [ ] A defined and predictive process that conforms to the principles of Scientific Management.
+- [ ] A complete methodology that defines how to develop software.
+- [ ] A cookbook that defines best practices for software development.
```
Comment (MrWolfbear) [inline README.md:3090]: image

File: README.md [hunk 3]
```diff
@@ -3060,4 +3071,113 @@ _[⬆ Back to Top](#table-of-contents)_
...
+
+_[⬆ Back to Top](#table-of-contents)_
+
+### The timebox for a Daily Scrum is? (choose the best answer)
+
+- [x] 15 minutes.
+- [ ] 15 minutes for a 4-week sprint. For shorter Sprints it is usually shorter.
+- [ ] Two minutes per person.
+- [ ] 4 hours.
+- [ ] The same time of day every day.
```
Comment (MrWolfbear) [inline README.md:3101]: image

File: README.md [hunk 4]
```diff
@@ -3060,4 +3071,113 @@ _[⬆ Back to Top](#table-of-contents)_
...
+
+
+_[⬆ Back to Top](#table-of-contents)_
+
+### The Developers should not be interrupted during the Sprint and the Sprint Goal should remain intact. These are conditions that foster creativity, quality and productivity. (Which one the following answers is FALSE?)
+
+- [ ] The Developers may work with the Product Owner to add or remove work if they find themselves with more or less capacity than expected.
+- [x] The Sprint Backlog is fully formulated in the Sprint Planning event and does not change during the Sprint.
+- [ ] As a decomposition of the selected Product Backlog items, the Sprint Backlog changes and may grow as the work emerges.
+- [ ] The Product Owner can help clarify or optimize the Sprint when asked by the Developers.
```
Comment (MrWolfbear) [inline README.md:3111]: image

File: README.md [hunk 5]
```diff
@@ -3060,4 +3071,113 @@ _[⬆ Back to Top](#table-of-contents)_
...
+
+_[⬆ Back to Top](#table-of-contents)_
+
+### The three pillars of empiricism are: (choose the best answer)
+
+- [ ] Respect For People, Kaizen, Eliminating Waste.
+- [ ] Planning, Inspection, Adaptation.
+- [ ] Planning, Demonstration, Retrospective.
+- [x] Inspection, Transparency, Adaptation.
+- [ ] Transparency, Eliminating Waste, Kaizen.
```
Comment (MrWolfbear) [inline README.md:3122]: image

File: README.md [hunk 6]
```diff
@@ -3060,4 +3071,113 @@ _[⬆ Back to Top](#table-of-contents)_
...
+
+
+_[⬆ Back to Top](#table-of-contents)_
+
+### What is the typical size for a Scrum Team? (choose the best answer)
+
+- [x] 10 or fewer.
+- [ ] Minimum of 7.
+- [ ] 7 plus or minus 2.
+- [ ] 9.
```
Comment (MrWolfbear) [inline README.md:3132]: image

File: README.md [hunk 7]
```diff
@@ -3060,4 +3071,113 @@ _[⬆ Back to Top](#table-of-contents)_
...
+
+
+_[⬆ Back to Top](#table-of-contents)_
+
+### What is the function or purpose of Management in Scrum? (choose the best answer)
+
+- [ ] Identify and remove people that are not working hard enough.
+- [ ] Monitor the progress of the Developers on the Scrum Team.
+- [x] Support the Product Owner with insights and information into high value product and system capabilities. Support the Scrum Master to encourage organizational change that fosters empiricism, self-management, bottom-up intelligence, and intelligent product delivery.
+- [ ] Continually monitor staffing levels of the Scrum Team.
```
Comment (MrWolfbear) [inline README.md:3142]: image

File: README.md [hunk 8]
```diff
@@ -3060,4 +3071,113 @@ _[⬆ Back to Top](#table-of-contents)_
...
+
+
+_[⬆ Back to Top](#table-of-contents)_
+
+### What is the main reason for the Scrum Master to be at the Daily Scrum? (choose the best answer)
+
+- [ ] To gather status and progress information to report to management.
+- [ ] To write down any changes to the Sprint Backlog, including adding new items, and tracking progress on the burn-down.
+- [x] They do not have to be there; they only need to ensure the Developers have a Daily Scrum.
+- [ ] To make sure every team member answers the three questions.
```
Comment (MrWolfbear) [inline README.md:3152]: image

File: README.md [hunk 9]
```diff
@@ -3060,4 +3071,113 @@ _[⬆ Back to Top](#table-of-contents)_
...
+
+_[⬆ Back to Top](#table-of-contents)_
+
+### Which two things should the Scrum Team do during the first Sprint? (choose the best two answers)
+
+- [x] Develop and deliver at least one piece of functionality.
+- [ ] Develop a plan for the rest of the release.
+- [x] Deliver an Increment of useful and valuable product.
+- [ ] Determine the complete architecture and infrastructure for the product.
+- [ ] Create the complete Product Backlog to be developed in subsequent Sprints.
```
Comment (MrWolfbear) [inline README.md:3163]: image

File: README.md [hunk 10]
```diff
@@ -3060,4 +3071,113 @@ _[⬆ Back to Top](#table-of-contents)_
...
+
+
+_[⬆ Back to Top](#table-of-contents)_
+
+### True or False: It is mandatory that the product Increment be released to production at the end of each Sprint.
+
+- [ ] True.
+- [x] False.
```
Comment (MrWolfbear) [inline README.md:3171]: image

File: README.md [hunk 11]
```diff
@@ -3060,4 +3071,113 @@ _[⬆ Back to Top](#table-of-contents)_
...
+
+_[⬆ Back to Top](#table-of-contents)_
+
+### When should a Developer on a Scrum Team be replaced? (choose the best answer)
+
+- [ ] As needed, with no special allowance for changes in productivity.
+- [ ] Every Sprint to promote shared learning.
+- [x] As needed, while taking into account a short-term reduction in productivity.
+- [ ] Never, it reduces productivity.
```
Comment (MrWolfbear) [inline README.md:3180]: image

**Comment by danieldanielecki** [comment]
Thanks, will take a look anytime soon!

**Comment by danieldanielecki** [comment]
It seems to me some of them mix from PSM I/PSPO I, but can be merged. These questions between PSD/PSM/PSPO are being used interchangeable.

**Pull request merged**